src/vos/tests/vts_io.c

```diff
@@ -817,7 +817,7 @@ io_obj_cache_test(void **state)
 	rc = vos_pool_create(po_name, pool_uuid, VPOOL_16M, 0);
 	assert_int_equal(rc, 0);
 
-	rc = vos_pool_open(po_name, pool_uuid, &l_poh);
+	rc = vos_pool_open(po_name, pool_uuid, false /* small */, &l_poh);
```

Comment:
what is considered small?   I see these warnings in vos tests too

Reply:
(roughly, for this discussion just considering what vos_space.c functions set aside for fragmentation overhead, and considering SCM size):

A vos pool that specifies 4GB or lower of SCM, due to the changes in PR 3066, will set aside a minimum of 2GB of SCM for fragmentation avoidance. The code calculates 5% of SCM size but bumps it to 2GB as a minimum requirement. This 2GB reserved memory is greater than half the size of the pool SCM capacity that will trigger vos_space_sys_init() to decide to disable space reservation altogether for the pool. Generally this may be a good idea, but for situations like rdb where the caller knows it needs some nominal fragmentation protection (basic "5% of default 1GB SCM size"), then it can specify that it is creating a "small" pool that should not have this 2GB minimum reserved memory threshold applied.